contrib/chatops/actions/format_execution_result.py

```diff
@@ -42,3 +43,12 @@ def run(self, execution):
                     template = alias.result['format']
 
         return self.jinja.from_string(template).render(context)
+
+    def _get_execution(self, execution_id):
+        if not execution_id:
+            raise ValueError('Invalid execution_id provided.')
+        execution = self.client.liveactions.get_by_id(id=execution_id)
```

Comment:
I checked the code, looks like the client name is wrong.

Gosh, this is so annoying and confusing. Maybe to make it less confusing let's create new `Execution` resource class in `st2client/models/action|execution` and let's have `LiveAction` inherit from it and leave it there for backward compatibility reasons.